src/hooks/useCustomAssets.ts

```diff
@@ -156,27 +163,27 @@ const useCustomAssets = () => {
         fetchAsset();
       }
     },
-    [fetchAsset, verifiedIbcAssets],
+    [fetchAsset, verifiedIbcAssets, fetchDecimal, hasChainPrefix],
```

Comment:
```suggestion
    [fetchAsset, verifiedIbcAssets, fetchDecimal],
```

It seems unnecessary